src/CBT.Core/Internal/ModulePropertyGenerator.cs

```diff
@@ -12,8 +12,10 @@ namespace CBT.Core.Internal
 {
     internal sealed class ModulePropertyGenerator
     {
-        internal static readonly string ImportRelativePath = Path.Combine("CBT", "Module", "$(MSBuildThisFile)");
-        internal static readonly string ModuleConfigPath = Path.Combine("CBT", "Module", "module.config");
+        internal static readonly string ImportRelativePath = Path.Combine("build", "$(MSBuildThisFile)");

```

Comment:
We need to not use `$(MSBuildThisFile)` anymore right?  This needs to be the `packageId.props` now instead?